pcsx2/x86/ix86-32/iR5900-32.cpp

```diff
@@ -1184,7 +1184,15 @@ void recMemcheck(u32 op, u32 bits, bool store)
 	xFastCall((void*)standardizeBreakpointAddress, ecx);
 	xMOV(ecx,eax);
 	xMOV(edx,eax);
-	xADD(edx,bits/8);
+
+	if ((psxHu32(HW_ICFG) & (1 << 3))) // if we're in PS1 mode adjust ee accordingly.
```

Comment:
Hum I far as I understand, bits is the number of bits that will be read for the memory access. So it will be 8, 16, 32, 64. The division by 8 give you a size in byte. I don't understand why you need to change this stuff for PSX.

Reply:
That was my bad, mistook it for a timing function, it wasn't. My bad :P

Reply:
It has to do with the frequency of IOP to EE even if it only returns size, before I made the change in that file and had only edited the interpreter multiplier recompiler was foobar by adjusting that it kinda started to work albeit with pgif errors

Reply:
I'm not even sure this code is used. It is related to debugger breakpoint.